### ISSUE TITLE
Pretrain on custom dataset without labels

### ISSUE THREAD

**Issue by ChintanTrivedi** [vturrisi/solo-learn#94]
I am looking to pretrain a model on images for which labels are not available/defined. For that, I'm trying to replicate the DataLoader for other datasets in the catalog (like cifar10), but it seems the methods are implemented to return labels along with the images. Could you please guide on how/what to tweak in order to make the training pipeline work without labels?

**Comment by vturrisi** [comment]
Hey,

At the moment, the easiest way would be to just return -1 for all labels. Those are ignored when training the linear classifier, so I think you will be good. I'm not sure, but an error might arise in the evaluation part of the code. If it does, I would simply create dummy labels for your data.

We are working on making an interface for custom datasets that will handle that.

**Comment by DonkeyShot21** [comment]
Support for custom datasets has been added in #103
